### PR TITLE
add cpu load to dataflash PM packet

### PR DESCRIPTION
**Pull request by kd0aij**
this is the same load value as in mavlink message SYS_STATUS.load
Also added % (percent) to the set of log units

### CODE REVIEW

**Comment by magicrub** [comment]
looks fine, would be nice on the other vehicles too. Lets get this merged and then copy/paste

File: ArduPlane/Log.cpp [hunk 1]
```diff
@@ -76,7 +77,8 @@ void Plane::Log_Write_Performance()
         g_dt_max        : perf.G_Dt_max,
         g_dt_min        : perf.G_Dt_min,
         log_dropped     : DataFlash.num_dropped() - perf.last_log_dropped,
-        hal.util->available_memory()
```
Comment (rmackay9) [inline ArduPlane/Log.cpp]: It's funny that this line doesn't have "mem_avail : " and "load :" at the front to specify the field.  I guess it works but it doesn't quite match the style we normally use.  Not saying this should stop the PR from going in but personally I'd fix that.

**Comment by rmackay9** [comment]
I'm happy with this going into copter.  I have made a comment about the field name being missing when the dataflash structure is being filled in but that doesn't necessarily need to be fixed before this goes in.

**Comment by kd0aij** [comment]
@rmackay9 Thanks for pointing that out; I didn't even notice that the last line in that struct was different when I copied its syntax. I'll fix it shortly.

**Comment by peterbarker** [comment]
It would be really nice if this went in first:  https://github.com/ArduPilot/ardupilot/pull/7223

Doing the same thing for every vehicle is a waste....

**Comment by kd0aij** [comment]
@peterbarker I agree; and was wondering why the performance message wasn't in common...
I should have looked through existing issues and PRs before doing this.

**Comment by amilcarlucas** [comment]
#7223 has been merged, can this be rebased ?

**Comment by kd0aij** [comment]
replaced by #7722

**Pull request closed without merging**